src/presentation/api/controller/ssl.go

```diff
@@ -46,27 +46,33 @@ func (controller *SslController) Read(c echo.Context) error {
 	return apiHelper.ServiceResponseWrapper(c, controller.sslService.Read())
 }
 
-func parseRawVhosts(rawVhostsInput interface{}) (rawVhosts []string, err error) {
-	rawVhostsSlice, assertOk := rawVhostsInput.([]interface{})
+func (controller *SslController) parseRawVhosts(
+	rawVhostsInput interface{},
+) ([]string, error) {
+	rawVhostsStrSlice, assertOk := rawVhostsInput.([]string)
```

Comment:
I added it to the method's return statement instead of initializing it at the start, since the value would never be used and this would trigger a warning from the **Go** linter.